.github/workflows/pr_test_build.yml

```
@@ -46,7 +46,7 @@ jobs:
           channel: stable
 
       - name: Install package dependencies
-        run: sudo apt-get install -y curl unzip automake build-essential file pkg-config git python libtool libtinfo5 cmake clang
+        run: sudo apt-get install -y curl unzip automake build-essential file pkg-config git python libtool libtinfo5 cmake clang libzmq3-dev zmq
```

```suggestion
        run: sudo apt-get install -y curl unzip automake build-essential file pkg-config git python libtool libtinfo5 cmake clang libzmq3-dev
```

i dont think there is a `zmq` package for ubuntu 20.04

The actions build fails with libwallet stuff being unavail. I'm not sure that is zmq related

---

yeah I noticed, but there are a lot of problems, so I am trying to see which one of them is the cause of failure. the first error was `Could not find required header zmq.h` so I added `libzmq3-dev`
but then it followed with the second error `Could not find required libzmq`
now I am thinking that this was not the issue, rather this error is the issue
```
fatal: remote error: upload-pack: not our ref 6405ac6e***708938ffd8631***9e187879bf74d***f3
2024-03-22T11:18:38.1938071Z fatal: Fetched in submodule path 'external/trezor-common', but it did not contain 6405ac6e***708938ffd8631***9e187879bf74d***f3. Direct fetching of that commit failed.
```

---

but at least there are some errors here that I can trace, meanwhile on iOS, it just fails 😭